doc/man3/EVP_EncryptInit.pod

```diff
@@ -107,6 +108,9 @@ EVP_CIPHER_do_all_ex
                     const unsigned char *key, const unsigned char *iv, int enc);
  int EVP_CipherFinal(EVP_CIPHER_CTX *ctx, unsigned char *outm, int *outl);
 
+ int EVP_Cipher(EVP_CIPHER_CTX *ctx, unsigned char *out,
```

Comment:
Are we going to deprecate this?

Reply:
I think that should be left to another PR.  This fixes a blatant bug, let's stay with that.